content/en/docs/concepts/security/pod-security-policy.md

```diff
@@ -12,736 +12,9 @@ weight: 30
-```
-
-```none
-deployment "pause" created
-```
-```shell
-kubectl-user get pods
-```
-
-```
-No resources found.
-```
-
-```shell
-kubectl-user get events | head -n 2
-```
-
-```
-LASTSEEN   FIRSTSEEN   COUNT     NAME              KIND         SUBOBJECT                TYPE      REASON                  SOURCE                                  MESSAGE
-1m         2m          15        pause-7774d79b5   ReplicaSet                            Warning   FailedCreate            replicaset-controller                   Error creating: pods "pause-7774d79b5-" is forbidden: no providers available to validate pod request
-```
-
-**What happened?** We already bound the `psp:unprivileged` role for our `fake-user`,
-why are we getting the error `Error creating: pods "pause-7774d79b5-" is
-forbidden: no providers available to validate pod request`? The answer lies in
-the source - `replicaset-controller`. Fake-user successfully created the
-deployment (which successfully created a replicaset), but when the replicaset
-went to create the pod it was not authorized to use the example
-podsecuritypolicy.
-
-In order to fix this, bind the `psp:unprivileged` role to the pod's service
-account instead. In this case (since we didn't specify it) the service account
-is `default`:
-
-```shell
-kubectl-admin create rolebinding default:psp:unprivileged \
-    --role=psp:unprivileged \
-    --serviceaccount=psp-example:default
-```
-
-```none
-rolebinding "default:psp:unprivileged" created
-```
-
-Now if you give it a minute to retry, the replicaset-controller should
-eventually succeed in creating the pod:
-
-```shell
-kubectl-user get pods --watch
-```
-
-```none
-NAME                    READY     STATUS    RESTARTS   AGE
-pause-7774d79b5-qrgcb   0/1       Pending   0         1s
-pause-7774d79b5-qrgcb   0/1       Pending   0         1s
-pause-7774d79b5-qrgcb   0/1       ContainerCreating   0         1s
-pause-7774d79b5-qrgcb   1/1       Running   0         2s
-```
-
-### Clean up
-
-Delete the namespace to clean up most of the example resources:
-
-```shell
-kubectl-admin delete ns psp-example
-```
-
-```
-namespace "psp-example" deleted
-```
-
-Note that `PodSecurityPolicy` resources are not namespaced, and must be cleaned
-up separately:
-
-```shell
-kubectl-admin delete psp example
-```
-
-```
-podsecuritypolicy "example" deleted
-```
-
-### Example Policies
-
-This is the least restrictive policy you can create, equivalent to not using the
-pod security policy admission controller:
-
-{{< codenew file="policy/privileged-psp.yaml" >}}
-
-This is an example of a restrictive policy that requires users to run as an
-unprivileged user, blocks possible escalations to root, and requires use of
-several security mechanisms.
-
-{{< codenew file="policy/restricted-psp.yaml" >}}
-
-See [Pod Security Standards](/docs/concepts/security/pod-security-standards/#policy-instantiation)
-for more examples.
-
-## Policy Reference
```

Comment:
Do you think the policy reference could be useful to people looking to recreate their PSPs elsewhere?

Reply:
historical content will be available at https://v1-24.docs.kubernetes.io/ … not sure we should keep info from non-current versions at HEAD